misc/bazel/internal/zipmerge/BUILD.bazel

```diff
@@ -20,4 +17,12 @@ cc_binary(
     ],
 )
 
-#TODO port tests from internal repo
+cc_test(
```

Comment:
Are we ensuring this still gets picked up by our internal tests? `//...` doesn't recurse into other modules.